ghost/core/core/server/services/email-analytics/lib/queries.js

```diff
@@ -14,30 +13,36 @@ module.exports = {
     async getLastSeenEventTimestamp() {
         const startDate = new Date();
 
-        // three separate queries is much faster than using max/greatest (with coalesce to handle nulls) across columns
-        let {maxDeliveredAt} = await db.knex('email_recipients').select(db.knex.raw('MAX(delivered_at) as maxDeliveredAt')).first() || {};
-        let {maxOpenedAt} = await db.knex('email_recipients').select(db.knex.raw('MAX(opened_at) as maxOpenedAt')).first() || {};
-        let {maxFailedAt} = await db.knex('email_recipients').select(db.knex.raw('MAX(failed_at) as maxFailedAt')).first() || {};
-
-        if (maxDeliveredAt && !(maxDeliveredAt instanceof Date)) {
-            // SQLite returns a string instead of a Date
-            maxDeliveredAt = new Date(maxDeliveredAt);
-        }
-
-        if (maxOpenedAt && !(maxOpenedAt instanceof Date)) {
-            // SQLite returns a string instead of a Date
-            maxOpenedAt = new Date(maxOpenedAt);
+        // First, try to fetch the max latest_event_timestamp
+        let {maxTimestamp} = await db.knex('emails').select(db.knex.raw('MAX(latest_event_timestamp) as maxTimestamp')).first() || {};
+
+        if (maxTimestamp) {
+            if (!(maxTimestamp instanceof Date)) {
+                maxTimestamp = new Date(maxTimestamp);
```

Comment:
Maybe worth explaining why we need to convert to a Date here?

// SQLite returns a string instead of a Date